templates/emails/collective.expense.approved.hbs

```diff
@@ -30,7 +30,7 @@ Subject: Your expense to {{collective.name}} for {{{currency expense.amount curr
   </a>
 
   <br /><br />
-  <a href="{{actions.viewLatestExpenses}}" class="btn"><div>View Latest Expenses</div></a>
+  <a href="{{actions.viewLatestExpenses}}" class="btn"><div>View Expense</div></a>
```

Comment:
Changing the label only makes sense if we change the link, the goal here is to redirect to the expense URL